Formula/htmlcleaner.rb

```diff
@@ -22,18 +22,12 @@ class Htmlcleaner < Formula
   def install
     ENV["JAVA_HOME"] = Formula["openjdk"].opt_prefix
 
-    inreplace "pom.xml" do |s|
-      # Homebrew's OpenJDK no longer accepts Java 5 source
-      # Reported upstream at https://sourceforge.net/p/htmlcleaner/bugs/235/
-      s.gsub! "<source>1.5</source>", "<source>1.8</source>"
-      s.gsub! "<target>1.5</target>", "<target>1.8</target>"
-      # OpenJDK >14 doesn't support older maven-javadoc-plugin versions
-      s.gsub! "<version>2.9</version>", "<version>3.2.0</version>"
-    end
+    # OpenJDK >14 doesn't support older maven-javadoc-plugin versions
+    inreplace "pom.xml", "<version>2.9</version>", "<version>3.2.0</version>"
 
     system "mvn", "clean", "package", "-DskipTests=true", "-Dmaven.javadoc.skip=true"
```

Comment:
On a side note, since we have `-Dmaven.javadoc.skip=true` I don't think the `inreplace` is actually needed. The build went fine locally.